NitroxServer/ConsoleCommands/StopCommand.cs

```diff
@@ -4,17 +4,18 @@
 
 namespace NitroxServer.ConsoleCommands
 {
-    internal class ExitCommand : Command
+    internal class StopCommand : Command
     {
         public override IEnumerable<string> Aliases { get; } = new[] { "exit", "halt", "quit" };
 
-        public ExitCommand() : base("stop", Perms.ADMIN, "Stops the server")
+        public StopCommand() : base("stop", Perms.ADMIN, "Stops the server")
```

Comment:
Again maybe a rename to shutdown or something as stop isn't too descriptive of what it does?